modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} router for dual-stack support
+
+When you install a {product-title} cluster on bare metal infrastructure, you can configure the router for single-stack or dual-stack support.
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on bare metal.
+* You installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+////
+. Open the Network YAML file by entering the following command:
++
+[source,terminal]
+----
+$ oc get network/cluster -o yaml
+----
+////
+
+. To configure the router for single-stack or dual-stack support, create a service:
```

Comment:
The router already supports single-stack and dual-stack services. If a customer wants to take advantage of this, they will need to create (or modify) a service with the IP families field set however they desire (modifying existing services to grant additional IP families is allowed, I believe, but deleting an IP family after creation is not, I think).

Reply:
Thank you for explaining this! I'm editing the procedure. Do you think it's necessary to call out that deleting an IP family after creation is not allowed?

Reply:
Possibly? The API would emit an error if a user tried to remove an IP family from a service, so might not have to call that out directly.